src/components/plugin-app.tsx

```diff
@@ -65,6 +65,10 @@ export default class PluginApp extends React.Component<IProps, IState> {
           wrappedEmbeddableDiv={wrappedEmbeddableDiv}
           wrappedEmbeddableContext={wrappedEmbeddableContext}
           logEvent={this.logEventMethod}
+          sendCustomMessage={pluginContext && pluginContext.wrappedEmbeddable &&
+            pluginContext.wrappedEmbeddable.sendCustomMessage
```

Comment:
It's a shame you're not using a version of TypeScript that supports optional chaining.

Reply:
Agreed.  We should upgrade it next time someone has a chance.  Probably a bunch of places we could save ourselves some of this.